clients/imodels-client-management/src/base/types/RestClient.ts

```diff
@@ -43,6 +43,20 @@ export interface HttpRequestParams {
   headers: Dictionary<string>;
 }
 
+/** Abstraction for accessing HTTP response headers that the server responded with. */
+export interface HttpResponseHeaders {
+  /** Gets HTTP response header's value by the specified {@link headerName}. */
```

Comment:
we should document that we expect the implementation to be case-insensitive

Reply:
yes, forgot that, thanks